.github/workflows/build-release.yml

```diff
@@ -0,0 +1,44 @@
+name: build-release
+
+on:
+  push:
+    branches:
+      - master
+    paths:
+      - package.json
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - id: setup-node
+        uses: actions/setup-node@v2.5.1
+        with:
+          node-version: 12
```

Comment:
Node version will be 16: https://github.blog/changelog/2022-09-22-github-actions-all-actions-will-begin-running-on-node16-instead-of-node12/